test/inductor/test_torchinductor.py

```diff
@@ -3851,8 +3851,12 @@ def fn(x, w):
         w = torch.randn([1, 1, 5, 5])
 
         torch._dynamo.mark_dynamic(x, 0)
-
-        self.common(fn, (x, w))
+        check_lowp = True
+        # TODO: fix this. OneDNN convolution with output channel == 1
```

Comment:
Can you create an issue and reference this issue here ? To have some more context why this is needed ?

Reply:
Hi, @atalman , actually it's it's a matter of precision. I've updated the code and also created a issue https://github.com/pytorch/pytorch/issues/129974 for it.